### DISCUSSION TITLE
Need ability to convert checkbox to issue in Projects

### DISCUSSION THREAD

**Discussion by DavidIsaacBPS** [community/community#86650] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Projects

### Body

The feature to convert a checkbox line to an issue works from the Issues page but not from the Projects page.  This should work the same way in both.

### Steps to reproduce:
* Github > Projects > select project > select issues > hover over checkbox > observe "Convert to issue" is NOT visible.
* Github > Issues > select issue > hover over checkbox > observe "Convert to issue" is visible.

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by DavidIsaacBPS** [comment]
Could be related to #4644 but there is not enough detail there to know.

**Comment by mishmanners** [comment]
Yes this is correct. You can only use the "Convert to Issue" from the issue itself, not the Project pane. You can also only convert to issue from the body of an issue not a comment. Ie. you can't create a list in comments and then convert an item in that comment's list to an issue.

**Reply by DavidIsaacBPS** [reply]
I updated the title to clarify that this is a needed feature.  Please let me know if this is in the wrong place.

**Reply by mishmanners** [reply]
Ahh yes, we love having feedback! This is also a feature I would love to have as well 😄 I'll tag Projects and the Projects team should be able to take a look at this feature request.
